--- a/sound/soc/soc-utils.c    2024-11-13 00:19:21.883046131 -0600
+++ b/sound/soc/soc-utils.c    2024-11-13 00:20:26.918794520 -0600
@@ -9,6 +9,7 @@
 
 #include <linux/platform_device.h>
 #include <linux/export.h>
+#include <linux/module.h>
 #include <linux/math.h>
 #include <sound/core.h>
 #include <sound/pcm.h>
@@ -247,11 +248,24 @@
 	return ret;
 }
 
+
+
+
+
+
+
+static const struct of_device_id soc_dummy_ids[] = {
+	{ .compatible = "linux,snd-soc-dummy", },
+	{}
+};
+MODULE_DEVICE_TABLE(of, soc_dummy_ids);
 static struct platform_driver soc_dummy_driver = {
 	.driver = {
 		.name = "snd-soc-dummy",
+		.of_match_table = of_match_ptr(soc_dummy_ids),
 	},
 	.probe = snd_soc_dummy_probe,
+	
 };
 
 static struct platform_device *soc_dummy_dev;
