--- a/arch/arm64/boot/dts/freescale/imx8mq-phanbell.dts	2022-06-06 21:34:46.000000000 -0500
+++ b/arch/arm64/boot/dts/freescale/imx8mq-phanbell.dts	2022-09-16 23:20:04.628684142 -0500
@@ -286,6 +286,12 @@
 	status = "okay";
 };
 
+&uart3 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart3>;
+	status = "okay";
+};
+
 &usdhc1 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc1>;
@@ -379,6 +385,13 @@
 		>;
 	};
 
+	pinctrl_uart3: uart3grp {
+		fsl,pins = <
+			MX8MQ_IOMUXC_UART3_RXD_UART3_DCE_RX		0x79
+			MX8MQ_IOMUXC_UART3_TXD_UART3_DCE_TX		0x79
+		>;
+	};
+
 	pinctrl_usdhc1: usdhc1grp {
 		fsl,pins = <
 			MX8MQ_IOMUXC_SD1_CLK_USDHC1_CLK			0x83
