--- a/arch/arm64/boot/dts/freescale/imx8mq-phanbell.dts	2024-10-29 14:32:19.915804440 -0500
+++ b/arch/arm64/boot/dts/freescale/imx8mq-phanbell.dtsb	2024-10-29 14:36:07.848427553 -0500
@@ -339,6 +339,17 @@
 };
 
 &iomuxc {
+
+	pinctrl_sai1: sai1grp {
+		fsl,pins = <
+				MX8MQ_IOMUXC_SAI1_TXFS_SAI1_TX_SYNC     0xd6
+				MX8MQ_IOMUXC_SAI1_TXC_SAI1_TX_BCLK      0xd6
+				MX8MQ_IOMUXC_SAI1_MCLK_SAI1_MCLK        0xd6
+				MX8MQ_IOMUXC_SAI1_TXD0_SAI1_TX_DATA0    0xd6
+				MX8MQ_IOMUXC_SAI1_RXD0_SAI1_RX_DATA0    0xd6
+			>;
+	};
+
 	pinctrl_fec1: fec1grp {
 		fsl,pins = <
 			MX8MQ_IOMUXC_ENET_MDC_ENET1_MDC			0x3
@@ -492,3 +503,12 @@
 		>;
 	};
 };
+
+&sai1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_sai1>;
+	#assigned-clocks = <&clk IMX8MQ_CLK_SAI1>;
+	#assigned-clock-parents = <&clk IMX8MQ_AUDIO_PLL1_OUT>;
+	#assigned-clock-rates = <24576000>;
+	status = "okay";
+};
\ No newline at end of file

