--- a/arch/arm64/boot/dts/freescale/imx8mq.dtsi	2024-10-04 13:50:18.219441909 -0500
+++ b/arch/arm64/boot/dts/freescale/imx8mq.dtsi 2024-10-04 13:50:47.679800517 -0500
@@ -317,6 +317,23 @@
 		interrupt-name = "irq_busfreq_0", "irq_busfreq_1", "irq_busfreq_2", "irq_busfreq_3";
 	};
 
+	sai1: sai@30010000 {
+		#sound-dai-cells = <0>;
+		compatible = "fsl,imx8mq-sai",
+			     "fsl,imx6sx-sai";
+		reg = <0x30010000 0x10000>;
+		interrupts = <GIC_SPI 95 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&clk IMX8MQ_CLK_SAI1_IPG>,
+			<&clk IMX8MQ_CLK_DUMMY>,
+			<&clk IMX8MQ_CLK_SAI1_ROOT>,
+			<&clk IMX8MQ_CLK_DUMMY>, <&clk IMX8MQ_CLK_DUMMY>;
+		clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3";
+		dmas = <&sdma2 8 1 0>, <&sdma2 9 1 0>;
+		dma-names = "rx", "tx";
+		fsl,dataline = <0 0xff 0xff>;
+		status = "disabled";
+	};
+
 	soc: soc@0 {
 		compatible = "simple-bus";
 		#address-cells = <1>;
@@ -338,22 +355,6 @@
 			#size-cells = <1>;
 			ranges = <0x30000000 0x30000000 0x400000>;
 
-			sai1: sai@30010000 {
-				#sound-dai-cells = <0>;
-				compatible = "fsl,imx8mq-sai",
-					     "fsl,imx6sx-sai";
-				reg = <0x30010000 0x10000>;
-				interrupts = <GIC_SPI 95 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clk IMX8MQ_CLK_SAI1_IPG>,
-					<&clk IMX8MQ_CLK_DUMMY>,
-					<&clk IMX8MQ_CLK_SAI1_ROOT>,
-					<&clk IMX8MQ_CLK_DUMMY>, <&clk IMX8MQ_CLK_DUMMY>;
-				clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3";
-				dmas = <&sdma2 8 1 0>, <&sdma2 9 1 0>;
-				dma-names = "rx", "tx";
-				fsl,dataline = <0 0xff 0xff>;
-				status = "disabled";
-			};
 
 			sai6: sai@30030000 {
 				#sound-dai-cells = <0>;
